src/App.scss

```diff
@@ -1,43 +1,10 @@
-@import "./theme/colors.scss";
-@import "./theme/typography.scss";
+@import "theme/colors";
+@import "theme/typography";
+@import "./components/Header";
+@import "./components/Profile";
+@import "./components/Feed";
 
-.App {
-  text-align: center;
-  background-color: yellow;
-}
-
-.App__logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
-}
-
-.App__header {
-  @include headline-text;
-
-  background-color: $gray-28;
-  min-height: 100vh;
+.Content__block {
```

Comment:
Is it weird to put this styling here? It matches the file format, but feels a little weird under the imports.  What's the right call?

Reply:
I'd likely name it `.App__content` instead, that way it follows more of BEM, but having this file as the App component's styles is fine by me.